src/imgtools/modules/interlacer.py

```
@@ -0,0 +1,180 @@
+        return isinstance(other, SeriesNode) and self.Series == other.Series
+
+    def __repr__(self, level=0):
+        """Recursive representation of the tree structure"""
+        indent = "  " * level
+        result = f"{indent}- {self.Modality}, (SERIES: {self.Series})\n"
+        for child in self.children:
+            result += child.__repr__(level + 1)
+        return result
+    
+class Branch:
+    def __init__(self, series_nodes: List[SeriesNode] = []):
```

Ruff (0.8.2)

Do not use mutable data structures for argument defaults

Replace with `None`; initialize within function

(B006)